CONTRIBUTING.rst

```diff
@@ -98,7 +117,9 @@ directory should contain the following files:
 **Tests** covering the functionality of the plugin should be added to the
 ``tests`` directory with the naming ``test_spamandeggs.py`` corresponsing to
 the plugin residing in the directory ``spamandeggs``. Data files for the tests
-should be placed in a corresponding subdirectory [change for pooch].
+should be placed in a corresponding subdirectory - see
+[Adding and Updating Test Data](#adding-and-updating-test-data) section for more
```

Comment:
Link not working in the doc-build of this PR: https://rosettasciio--123.org.readthedocs.build/en/123/contributing.html

Reply:
Markdown syntax in a `rst` file! 🤦‍♂️